apps/src/codebridge/InfoPanel/ValidatedInstructions.tsx

```diff
@@ -74,7 +75,14 @@ const ValidatedInstructions: React.FunctionComponent<InstructionsProps> = ({
   handleInstructionsTextClick,
   className,
 }) => {
-  const {onRun, onStop, levelProperties} = useCodebridgeContext();
+  const {
+    onRun,
+    onStop,
+    levelProperties,
+    getAiTutor2FullQuestionFromQuestion,
+    aiTutor2HintQuestion,
+    levelProperties: {aiTutor2Available},
```

Comment:
nit: `levelProperties` is already extracted above, can we just use that (and destructure below)?

Reply:
Good catch.  Updated.